PatientRegistrationApp/src/components/HouseholdContactInfo.vue

```diff
@@ -132,11 +117,8 @@ import EventBus from '../eventBus'
 			secondaryPhoneNumber: '',
 			secondaryPhoneNumberType: '',
 			secondaryEmail: '',
-			approval: '',
-			phoneNumberAvailable: true,
-			phoneNumberAvailableRadioButtons: 'yes',
-			emailAvailable: true,
-			emailAvailableRadioButtons: 'yes',
+			acknowledgement: false,
```

Comment:
Just looking over all the code, is `acknowledgment` and `permission` specific enough to what the variable means or could they be more specific? (sorry feel like this has been brought up before)

Reply:
Ok!  I changed them to acknowledgementCheckBox and permissionCheckBox, respectively